activities/TankOp.activity/play.js

```diff
@@ -57,6 +57,7 @@ enyo.kind({
 	create: function() {
 		this.inherited(arguments);
 		play = this;
+		pause = this;
```

Comment:
What's the objective of this line?

Reply:
@llaske I am sorry about that it was just a mistake I removed it and pushed it again.